server/app.mjs

```diff
@@ -34,38 +37,54 @@ export const startAppServer = ({dev, hostname, port, dir}) => {
         });
         const handler = app.getRequestHandler();
 
-        app.prepare().then(() => {
-            const httpServer = createServer(handler);
-            const io = new Server(httpServer);
-            const gptscript = new GPTScript();
-
-            io.on("connection", (socket) => {
-                io.emit("message", "connected");
-                socket.on("run", async (location, tool, args, scriptWorkspace, threadID) => {
-                    if (runningScript) {
-                        await runningScript.close();
-                        runningScript = null;
-                    }
-                    try {
-                        dismount(socket);
-                        await mount(location, tool, args, scriptWorkspace, socket, threadID, gptscript);
-                    } catch (e) {
-                        socket.emit("error", e);
-                    }
+        const gptscript = new GPTScript();
```

Comment:
Should we be passing the `DefaultModelProvider` here?

Reply:
yes, I'm including that change in this PR, but can move it here if you'd like: https://github.com/gptscript-ai/ui/pull/43/files#diff-faaf629f315a73df9ddcba2406a8c1a65c0f3e0f4b877241dae878a23563c89fR27

Reply:
Nope, all good.